packages/flutter_test/lib/src/test_compat.dart

```diff
@@ -144,7 +144,7 @@ Future<void> _runSkippedTest(Suite suiteConfig, Test test, List<Group> parents,
 ///       // This test is especially slow on Windows.
 ///       'windows': Timeout.factor(2),
 ///       'browser': [
-///         Skip('TODO: add browser support'),
+///         Skip('add browser support'),
```

Comment:
@jonahwilliams

Reply:
this is a doc comment example code block

Reply:
I understand this does not really matter to our todo tech-debts, but my question is do we really need a `TODO:` here? the sentence itself should be enough?

Reply:
fair enough